### ISSUE TITLE
Better documentation around passing options to Kroki

### ISSUE THREAD

**Issue by jaskij** [asciidoctor/asciidoctor-kroki#449]
I came here to open an issue similar to #418. Reading through the comments under it, specifically https://github.com/asciidoctor/asciidoctor-kroki/issues/418#issuecomment-2039249262, it turns out that unknown named attributes are just passed through to Kroki as diagram options.

Perhaps this is present in the README, but never stated explicitly.

**Comment by ggrossetie** [comment]
> I came here to open an issue similar to https://github.com/asciidoctor/asciidoctor-kroki/issues/418. Reading through the comments under it, specifically https://github.com/asciidoctor/asciidoctor-kroki/issues/418#issuecomment-2039249262, it turns out that unknown named attributes are just passed through to Kroki as diagram options.

> Perhaps this is present in the README, but never stated explicitly.

I agree, we should add this information. Feel free to open a pull request to add a section about it in the README.

**Comment by ggrossetie** [comment]
Fixed in https://github.com/asciidoctor/asciidoctor-kroki/commit/84f58316154bd3c9505b8ed4f0af1416b480e5fb
